scripts/webpack.scripts.js

```
@@ -37,8 +37,8 @@ module.exports = mergeWithShared({
   },
   resolve: {
     alias: {
-      "@/icons/list": path.resolve("src/__mocks__/iconsListMock"),
-      "@uipath/robot": path.resolve("src/__mocks__/robotMock"),
+      "@/icons/list": path.resolve("src/__mocks__/@/icons/list"),
+      "@uipath/robot": path.resolve("src/__mocks__/@uipath/robot"),
       "@/telemetry/reportError": path.resolve("src/__mocks__/reportError"),
```

I had also tried to auto-mock this one, but it's the opposite of our expected usage of mocks here: it breaks when called as opposed to silencing errors.